src/transaction/log_comm.c

```diff
@@ -269,6 +269,13 @@ log_dump_log_info (const char *logname_info, bool also_stdout, const char *fmt,
 bool
 log_does_allow_replication (void)
 {
+  return false;
+
+#if 0
+/* TODO - newHA
```

Comment:
Why is it TODO? Is this TODO for the NewHA?

Reply:
I had to leave it as a TODO because the DR concept was not decided. 
This is because generating this replication log may become necessary again for DR (what CTO said). 
I left it as a TODO, considering it part of completing the newHA, which includes implementing the DR concept